test/tbb/test_arena_priorities.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright (c) 2020-2021 Intel Corporation
+    Copyright (c) 2020-2022 Intel Corporation
```

Comment:
Seems like changes in this file should be discarded

Reply:
The empty line was removed :)

Reply:
The last change in the file was removed, so lets roll back the copyright and exclude the whole file from the patch